packages/examples/src/opentelemetry.ts

```diff
@@ -0,0 +1,23 @@
+import './instrument.js';
+import './simple-completion.js';
+/*
+This demo shows how tree rendering can be traced with OpenTelemetry. To view/collect traces with Jaeger you can use:
+
+docker run -d --name jaeger \
```

Comment:
Do we need to tell people what version of Docker this is, etc? Is this command fully self-contained?

Reply:
It's fully self-contained.